revista-latinoamericana-de-recursos-naturales.csl

```diff
@@ -0,0 +1,435 @@
+        <group>
+          <text variable="volume"/>
+          <text variable="issue" prefix="(" suffix=")"/>
+          <text variable="page" prefix=": "/>
+        </group>
+      </if>
+      <else-if type="article-newspaper">
+        <label variable="page" form="short" suffix=" "/>
+        <text variable="page"/>
+      </else-if>
+      <else>
+        <text term="volume" form="short" suffix=" "/>
+        <number variable="volume" form="numeric"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="number-of-volumes">
+    <text term="volume" form="short" plural="true" suffix=". "/>
+    <text variable="number-of-volumes"/>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher"/>
+  </macro>
+  <macro name="accessed">
+    <choose>
+      <if variable="URL">
+        <group prefix=" [" suffix="]" delimiter=":&#160;">
+          <text term="accessed" text-case="capitalize-first"/>
+          <date variable="accessed">
+            <date-part name="month" suffix=" "/>
+            <date-part name="day" suffix=", "/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <group delimiter=", " prefix="(" suffix=")">
+      <text variable="collection-title" prefix="ser. "/>
+      <text variable="collection-number" prefix="no. ser. "/>
+    </group>
+  </macro>
+  <macro name="page">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper post-weblog" match="any">
+        <text variable="page" prefix=": "/>
+      </if>
+      <else-if type="report" match="any">
+        <text variable="page" suffix=" p."/>
+      </else-if>
+      <else>
+        <label variable="page" form="short" suffix=" "/>
+        <text variable="page"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="number-of-pages">
+    <text variable="number-of-pages"/>
+    <text term="number-of-pages" form="short" prefix=" "/>
+  </macro>
+  <macro name="isbn">
+    <text variable="ISBN" prefix="ISBN: "/>
+  </macro>
+  <macro name="issn">
+    <text variable="ISSN" prefix="ISSN: "/>
+  </macro>
+  <macro name="doi">
+    <text variable="DOI" prefix="DOI: "/>
+  </macro>
+  <macro name="url">
+    <choose>
+      <if variable="URL">
+        <group>
+          <text term="retrieved" text-case="capitalize-first"/>
+          <text variable="URL" prefix=": &lt;" suffix="&gt;"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="note">
+    <text variable="note" prefix=" "/>
+  </macro>
+  <macro name="number">
+    <text variable="number" prefix="no. "/>
+  </macro>
+  <macro name="call-number">
+    <text variable="call-number" prefix="call no. "/>
+  </macro>
+  <macro name="authority">
+    <text variable="authority"/>
+  </macro>
+  <macro name="version">
+    <text variable="version" prefix="version "/>
+  </macro>
+  <macro name="section">
+    <text variable="section"/>
+  </macro>
```

Comment:
@rcervantesb, I just merged the style. Thanks!

If you do any more work on this style, note that it would be better to remove all these macros with one-liners, which just complicate the style. E.g. you can just replace

```xml
  <macro name="section">
     <text variable="section"/>
   </macro>
...
  <text macro="section" prefix="sect. "/>
```

by

```xml
  <text variable="section" prefix="sect. "/>
```